src/porcelain/green_node.jl

```diff
@@ -118,6 +118,27 @@ function Base.show(io::IO, ::MIME"text/plain", node::GreenNode, str::AbstractStr
     _show_green_node(io, node, "", 1, str, show_trivia)
 end
 
+function _show_green_node_sexpr(io, node::GreenNode, position)
+    if is_leaf(node)
+        print(io, position, "-", position+node.span-1, "::", untokenize(head(node); unique=false))
+    else
+        print(io, "(", untokenize(head(node); unique=false))
+        first = true
```

Comment:
This variable doesn't seem to be used at all?

Reply:
Thanks, fixed.   Also fixed it in the `SyntaxNode` code this was derived from.